### PR TITLE
Add proposals.adoc

### PR DESCRIPTION
**Pull request by cbeams**
This doc captures the clarifications and refinements to the proposals
process proposed in bisq-network/proposals#11.

### CODE REVIEW

File: proposals.adoc [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Create a new GitHub issue in the `bisq-network/proposals` repository containing the text of your proposal, written and formatted per the guidelines below.
+
+A maintainer will quickly review your proposal and will either (a) assign it to you to indicate your ownership and responsibility over it, or (b) close it and label it as `was:incorrect` if it does not follow the guidelines below.
+
+=== Step 2. Review
+
+Once a proposal is submitted, a two-week review period follows. During this period, interested reviewers should read, discuss and ultimately react to the proposal as follows:
+
+ - 👍: I agree with the proposal and want to see it enacted
```
Comment (blabno) [inline proposals.adoc]: This renders as square. I think something else was intended here.

Comment (cbeams) [reply proposals.adoc]: Interesting, looking into it.

Comment (cbeams) [reply proposals.adoc]: This is what I see in the deploy preview at https://deploy-preview-40--bisq-docs.netlify.com/proposals.html:

image

Can you share a screenshot of what you see?

Comment (blabno) [reply proposals.adoc]: proposals

Comment (mrosseel) [reply proposals.adoc]: screen shot 2018-04-23 at 15 03 28

Comment (blabno) [reply proposals.adoc]: I guess I miss some font.

**Pull request merged**